### PR TITLE
internal/debug: remove deprecated flags

### PR DESCRIPTION
**Pull request by Zachinquarantine**
The deprecated flags are removed in the main portion of geth, this PR removes it internally too.

### CODE REVIEW

**Comment by karalabe** [comment]
The code doesn't compile.

**Comment by Zachinquarantine** [comment]
This PR now also removes the legacy --debug and legacy --backtrace flag, since travis didn't like me leaving that in.

**Comment by Zachinquarantine** [comment]
> 
> 
> The code doesn't compile.

it now passes!

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM

One year should be enough for those debug flags imo

**Comment by Zachinquarantine** [comment]
> LGTM

can this please be merged then?

**Comment by MariusVanDerWijden** [comment]
@Zachinquarantine We're currently preparing a hotfix release, until this is done we can't merge any other PR's. Your PR will be included in 1.10.9

File: internal/debug/flags.go
Comment (rjl493456442) [inline internal/debug/flags.go:148]: There are two `ctx.GlobalString(backtraceAtFlag.Name)`

File: internal/debug/flags.go
Comment (rjl493456442) [inline internal/debug/flags.go:157]: There are two `legacyMemprofilerateFlag.Name`

**Comment by arslan-raza-143** [comment]
Good

**Pull request merged**
